auth/index.js

```diff
@@ -27,7 +27,6 @@ passport.deserializeUser((user, done) => {
 });
 
 function isAuthenticated(req) {
-  console.log(req.user);
```

Comment:
Ah yep, I see both of these were later moved (let the record reflect that I anticipated that in my first comment)